fda/datasets/_real_datasets.py

```diff
@@ -1,6 +1,5 @@
 import numpy as np
 import rdata
```

Comment:
The `rdata` package is only used in `fetch_cran_dataset`, is there any reason to let the import in the module instead to moved it into the function, as `skdatasets`?

Reply:
It is recommended to have the imports at the beginning unless there are powerful reasons for not doing that.